code/game/gamemodes/miniantags/abduction/abduction_gear.dm

```diff
@@ -437,7 +437,7 @@ Congratulations! You are now trained for xenobiology research!"}
 		species = "<span clas=='notice'>[H.dna.species.name]</span>"
 		if(L.mind && L.mind.changeling)
 			species = "<span class='warning'>Changeling lifeform</span>"
-		var/obj/item/organ/gland/temp = locate() in H.internal_organs
+		var/obj/item/organ/heart/gland/temp = locate() in H.internal_organs
```

Comment:
With the removal of the `heart_attack` var, having a cardiac arrest is now just caused by the absence of a beating heart. So in order for ayy surgery not to kill them, we need to make the organs a heart. This also fixes a problem where ayys could stick a whole bunch of organs in the same guy.